packages/parser/lib/css_parser.mly

```diff
@@ -276,7 +380,7 @@ nth_payload:
   /* 2n */
   | a = DIMENSION { Nth (AN (int_of_string (fst a))) }
   /* 2n-1 */
-  | a = DIMENSION WS? combinator = COMBINATOR b = NUMBER {
+  | a = DIMENSION WS? combinator = combinator b = NUMBER {
```

Comment:
I don't think this is correct. 2n-1 doesn't accept `>`

Reply:
You're right, thanks!